paths/job_locales/create.yaml

```diff
@@ -73,4 +73,20 @@ requestBody:
               type: string
             example:
             - abcd1234cdef1234abcd1234cdef1234
-x-cli-version: '2.5'
+          translation_team_ids:
```

Comment:
I think this should be `translator_team_ids` instead of `translation_team_ids` right?

Reply:
Yes, of course! Will fix.